src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.jsx

```diff
@@ -286,7 +285,19 @@ const GroupDetails = createReactClass({
             <LoadingIndicator />
           </PageContent>
         ) : (
-          this.renderContent(showGlobalHeader)
+          <Projects orgId={organization.slug} slugs={[project.slug]}>
+            {({projects, initiallyLoaded, fetchError}) =>
+              initiallyLoaded ? (
+                fetchError ? (
+                  <LoadingError message={t('Error loading the specified project')} />
+                ) : (
+                  this.renderContent(showGlobalHeader, projects[0])
```

Comment:
Will it ever be possible that projects.length == 0?

Reply:
I dont think so because the <Projects> util component always gives back a projects array with atleast the slugs populated like [{slug: 'javascript'}]